Makefile

```diff
@@ -52,7 +52,7 @@ build: kill
 	$(DOCKER_COMPOSE_CMD) build --pull diet-error-handler publishing-e2e-tests $(APPS_TO_BUILD)
 
 setup_dependencies:
-	$(DOCKER_COMPOSE_CMD) up -d elasticsearch6 mongo mysql postgres rabbitmq redis
+	$(DOCKER_COMPOSE_CMD) up -d elasticsearch6 mongo mongo-2.6 mysql postgres rabbitmq redis
```

Comment:
Should we suffix existing mongo to be 3.4 for consistency?

Reply:
That's one thing we could do. On the other hand if we intend at some point to upgrade MongoDB to version `3.6` (to take advantage of change streams) then we could probably bump all the apps to use this version and only then need to define that single version, at which point we could just call it `mongo`?

Reply:
I imagine it was a mistake in the first place to not suffix them with versions - that'd have probably made everything a bit easier and it's something we now do in govuk-docker: https://github.com/alphagov/govuk-docker/blob/2a7cc277761ca2fab0860abc7ccac4ebf9904edc/docker-compose.yml#L8 (where, horrifically, we seem to be using 3.6 mongo 😢 )

I think this just leaves a bit of confusion of why one suffixed but not the other, but no biggie - hopefully this test suite is retired before that genuinely becomes a problem.